spec/features/users/user_can_comment_on_question_spec.rb

```diff
@@ -3,6 +3,7 @@
 describe 'when a user visits a question show page' do
```

Comment:
Excellent. I see some other redundancies in our code that we can refactor later. I made a few cards about this on Waffle.io